receiver/cloudfoundryreceiver/converter.go

```diff
@@ -41,15 +47,34 @@ func convertEnvelopeToMetrics(envelope *loggregator_v2.Envelope, metricSlice pme
 	}
 }
 
+func convertEnvelopeToLogs(envelope *loggregator_v2.Envelope, logSlice plog.LogRecordSlice, startTime time.Time) error {
+	log := logSlice.AppendEmpty()
+	log.SetTimestamp(pcommon.Timestamp(envelope.GetTimestamp()))
+	log.SetObservedTimestamp(pcommon.NewTimestampFromTime(startTime))
+	logLine := string(envelope.GetLog().GetPayload())
+	log.Body().SetStr(logLine)
+	//exhaustive:enforce
+	switch envelope.GetLog().GetType() {
+	case loggregator_v2.Log_OUT:
```

Comment:
The general logic of logs sent to `stdout` being `INFO` and `stderr` being whatever we decide should probably be documented in the README.